charm/kafka/lib/charms/layer/kafka.py

```diff
@@ -169,7 +169,7 @@ def get_ip_for_interface(self, network_interface):
             try:
                 ip = netifaces.ifaddresses(network_interface)[2][0]['addr']
             except ValueError:
-                raise BigtopError(
+                raise Exception(
```

Comment:
We don't actually use get_ip_for_interface. Could remove this function entirely, in addition to all references to a 'network_interface' config option which no longer exists. Here or followup is fine.